Dockerfile

```diff
@@ -58,22 +51,15 @@ FROM alpine:latest
 
 LABEL Description="vzlogger"
 
-RUN \
-    echo "http://dl-cdn.alpinelinux.org/alpine/latest/main" >> /etc/apk/repositories && \ 
-    echo "http://dl-cdn.alpinelinux.org/alpine/latest/community" >> /etc/apk/repositories
-
 RUN apk add --no-cache \
     libcurl \
     gnutls \
     libsasl \
     libuuid \
-    # libssl1.1 \
     libgcrypt \
     libmicrohttpd \
-    # libltdl7 \
     json-c \
     libatomic \
-    # liblept5 \
     mosquitto-libs \
     libunistring \
     libstdc++6 \
```

Comment:
please delete libstdc++6. It is not needed, and not available on riscv64 / alpine:edge

Reply:
Done